tests/test_main.py

```diff
@@ -30,6 +32,11 @@ def test_run_main_error(self, mock_subprocess):
                      "http://test.com/api/v1/", False)
             self.assertIn("Must quit Xcode before writing to core data", out.getvalue())
 
+    def test_clear_generated_code_files(self):
+        self.assertTrue(os.path.isdir(BaseGenerator.BUILD_DIR))
```

Comment:
this looks like it depends on another test creating the build dir, my only concern is that if these tests were ran in a different order or moved around this would fail